Quotient/room.h

```diff
@@ -806,6 +801,8 @@ class QUOTIENT_API Room : public QObject {
 
     QJsonArray exportMegolmSessions();
 
+    void newSession(const QString& newSession);
```

Comment:
I'd rather use `newMegolmSession()`, so that there is less ambiguity with verification sessions. The same goes the other way around, e.g. `CryptoMachine` has `sessions` which actually are verification sessions, not Olm sessions.